node/core/pvf/Cargo.toml

```diff
@@ -36,18 +36,3 @@ adder = { package = "test-parachain-adder", path = "../../../parachain/test-para
 halt = { package = "test-parachain-halt", path = "../../../parachain/test-parachains/halt" }
 hex-literal = "0.3.1"
 tempfile = "3.2.0"
-
-# PVF execution leverages compiled artifacts provided by wasmtime. The contents of the artifacts
-# depends on the version of wasmtime. In this crate we persist the artifacts on disk so we should
-# be careful about the updates. In order to handle this, we depend on the wasmtime version here
-# that we think is used by the sc-executor. If wasmtime is updated in Substrate and wasn't updated
-# here then there will be linking errors like
-#
-#     `multiple definitions of `set_vmctx_memory`.
-#
-# or similar, because wasmtime exports these symbols and does not support multiple versions compiled
-# in at the same time.
-#
-# Another safeguard is a test `ensure_wasmtime_version` that will fail on each bump and prompt the
-# developer to correspondingly act upon the change.
-wasmtime-jit = "0.24"
```

Comment:
This is now removed along with the test before because as per https://github.com/paritytech/polkadot/pull/3084 we no longer leave artifacts in the cache and remove them each start. That means we do not need to foresee a possibility of an old artifact getting into the new engine.